temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid leaving debugging statements like `import pdb` in the code. These should be removed before committing.
```
Remove the line `import pdb`.
```